server/controllers/lettersControllers/lettersControllers.js

```diff
@@ -77,8 +77,8 @@ exports.createLetters = async (req, res) => {
 
 exports.updateLetters = async (req, res) => {
     try {
-        const {id} = req.params
-        const {quantity} = req.body
+        const { id } = req.params
```

Comment:
Are these just for readability, or are the spaces required? I saw that this was done on the createLetters as well

Reply:
Yeah, it's just essentially for readability.